StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/StandardTextDiagnosticBase.cs

```diff
@@ -96,11 +95,7 @@ protected static MatchResult HandleDeclaration(SyntaxNodeAnalysisContext context
                 var summaryNodes = summaryElement.Nodes().ToList();
                 if (summaryNodes.Count >= 3)
                 {
-                    var firstTextPartNode = summaryNodes[0] as XText;
-                    var classReferencePart = summaryNodes[1] as XElement;
-                    var secondTextPartNode = summaryNodes[2] as XText;
-
-                    if (firstTextPartNode != null && classReferencePart != null && secondTextPartNode != null)
+                    if (summaryNodes[0] is XText firstTextPartNode && summaryNodes[1] is XElement classReferencePart && summaryNodes[2] is XText secondTextPartNode)
```

Comment:
A couple more parenthesis will improve readability quite a bit here.

Reply:
❓ Is it sufficient to break the clauses to one clause per line, or are you specifically interested in parentheses?